C/services/notification/notification_manager.cpp

```diff
@@ -1400,10 +1400,10 @@ bool NotificationManager::setupRuleDeliveryFirst(const string& name, const Confi
 			if (deliver->ingestData())
 			{
 				
-				AssetTrackInfo* trakeringInfo = new AssetTrackInfo;
-				trakeringInfo->service = m_service;
-				trakeringInfo->notificationInstanceName = notificationName;
-				trakeringInfo->pluginName = deliver->getName();
+				AssetTrackInfo* trackingInfo = new AssetTrackInfo;
```

Comment:
make sure that the trackinginfo is correctly deleted.

Reply:
It is passed to callback function, that's why can't be deleted otherwise callback function may crash